src/CribblyBackend.DataAccess/Repositories/TeamRepository.cs

```diff
@@ -21,7 +22,32 @@ public TeamRepository(IDbConnection connection)
         {
             _connection = connection;
         }
+        public async Task<List<Team>> GetAll()
+        {
+            var players = new List<Player>();
+            var teams = (await _connection.QueryAsync<Team, Player, Team>(
+                @"SELECT * FROM Teams t INNER JOIN Players p ON t.Id = p.TeamId",
+                (t, p) => 
+                {
+                    switch (players.Count)
+                    {
+                        case 0:
+                            players.Add(p);
+                            break;
+                        case 1:
+                            players.Add(p);
+                            t.Players = new List<Player>();
+                            t.Players.AddRange(players);
+                            players.Clear();
+                            break;
+                    }
```

Comment:
What happens if there is something other than 0 or 1 player?
:bulb: Would it be better to keep a dictionary of team IDs to player lists and assign each team its players outside of this anonymous function before returning `teams`?

Reply:
How would we associate each player with their respective teams? Key values in dictionaries have to be unique

Reply:
It would be a `Dictionary<string, List<Player>>`; the team IDs are unique. Then after the query we can loop through the teams and grab the list of players for that team from the dictionary and assign it to the team.

Reply:
Made some changes - let me know if that works!